doc/nrf/ug_nrf5340.rst

```diff
@@ -151,7 +151,10 @@ For IEEE 802.15.4, the nRF Connect SDK makes use of Zephyr's :ref:`zephyr:nrf-ie
 This Zephyr sample is designed specifically to enable the nRF IEEE 802.15.4 radio driver and its serialization library on a remote MCU (for example, the nRF5340 network core) using the `RPMsg Messaging Protocol`_ as a transport for the nRF 802.15.4 radio driver serialization.
 The sample implements the RPMsg transport using the `OpenAMP`_ library to communicate with the nRF IEEE 802.15.4 radio driver serialization host that runs on a separate core (for example, the nRF5340 application core).
 
-This sample must be programmed to the network core to run samples for 802.15.4-based protocols like Thread or Zigbee on nRF5340.
+Thread and Zigbee
```

Comment:
I don't think this should be a subsection. Would there be any other subsections here?
I like Krzysztof's suggestion to add Thread and Zigbee to the IEEE 802.15.4 heading.

Reply:
What about "IEEE 802.15.4 (Thread and Zigbee)"?
I'm not a fan of backslashes in headings, and this option allows us to use subsections if more protocols start using 802.15.4.

Reply:
Sure, sounds good.